tests/rotation.py

```diff
@@ -22,28 +26,112 @@ def test_suspend_primary(network, args):
     return network
 
 
+@reqs.description("Suspend and resume all nodes while submitting constant client load")
+@reqs.at_least_n_nodes(3)
+def test_rolling_disconnections(network, args):
```

Comment:
This was an attempt at writing a test which repros the issue more consistently than repeated calls to `test_suspend_primary`. In practice its still a probabilistic repro, in an idiosyncratic and expensive test. I'm going to remove it since we now have a better test in `committable.py`. Long-term, we should probably run some suite of tests while separately driving load on the service via `vegeta`, but we shouldn't apply that load per-test like this.